lib/virtfs/context.rb

```diff
@@ -127,13 +166,23 @@ def chdir(dir)
       fs.dir_chdir(path) if fs.respond_to?(:dir_chdir)
     end
 
+    # @return [String] current filesystem working directory
     def getwd
       @cwd
     end
 
-    #
     # Expand symbolic links and perform mount indirection look up.
     #
+    # @param path [String] path to lookup
+    # @param raise_full_path [Boolean] indicates if error should be raised if lookup fails
+    # @param include_last [Boolean] indicates if last path component should be returned
+    #
+    # @raise [RunTimeError] if path could not be looked up and raise_full_path is true
+    # @raise [SystemCallError] if path could not be looked up
+    #
+    # @see #mount_lookup
+    # @see #expand_links
+    #
     def path_lookup(path, raise_full_path = false, include_last = true)
```

Comment:
The methods defined from here to the end of file are not part of the public VirtFS interface (they are for internal use). I'm not sure if they should be included in the documentation - or minimally, it should be made clear they are for internal use only.